src/supy/data_model/surface.py

```diff
@@ -360,7 +367,7 @@ def from_df_state(
             "sfr",
             "emis",
             "chanohm",
-            "cpanohm",
+            "rho_cpanohm",
             "kkanohm",
```

Comment:
let's change this to `k_anohm` – `kk` was originally used as a quick way to distinguish it from `k` in ESTM. Now, with the `anohm` suffix, we can adopt cleaner naming.